fetch.bs

```diff
@@ -5004,6 +5041,13 @@ steps:
 
      <li><p>Let <var>type</var> be <var>blob</var>'s {{Blob/type}}.
 
+     <li><p>Let <var>accept-coding</var> be the result of <a for="header list">getting</a>
+       `<code>Accept-Encoding</code>` from <var>request</var>'s <a for=request>header list</a>.
+        If <var>accept-encoding</var> is not null and the server selects one of the encoding options,
+        let <var>coding</var> be the selected encoding option; otherwise, i.e., no encoding is used,
```

Comment:
You can't have "let" in the middle of a phrase, see style in the rest of the document.